<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagSupporter</class>
 <widget class="QMainWindow" name="TagSupporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TagSupporter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1" rowspan="2" colspan="3">
     <widget class="QGroupBox" name="WebBrowsingGroup">
      <property name="title">
       <string>Web Browsing</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="WebBrowserWidget" name="webBrowserWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="PlaybackGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Playback</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PlaybackWidget" name="playbackWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="7">
     <layout class="QVBoxLayout" name="FileHandlingLayout">
      <item>
       <widget class="QGroupBox" name="FileBrowsingGroup">
        <property name="title">
         <string>File Browsing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="folderFileList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>save changes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="nextButton">
             <property name="text">
              <string>next</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>delete file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="folderContentLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="browseFolderButton">
             <property name="text">
              <string>Browse ...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refreshButton">
             <property name="text">
              <string>refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="CollectionBrowsingGroup">
        <property name="title">
         <string>Collection Browsing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="AmarokDatabaseWidget" name="amarokDatabaseWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="5">
     <widget class="QGroupBox" name="MetaDataGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Metadata</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MetadataWidget" name="metadataWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3" rowspan="5">
     <widget class="QGroupBox" name="WebSearchGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Websearch</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="OnlineSourcesWidget" name="onlineSourcesWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="4">
     <widget class="QGroupBox" name="FilenameGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Filename</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="FilenameWidget" name="filenameWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OnlineSourcesWidget</class>
   <extends>QWidget</extends>
   <header>OnlineSourcesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlaybackWidget</class>
   <extends>QWidget</extends>
   <header>PlaybackWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MetadataWidget</class>
   <extends>QWidget</extends>
   <header>MetadataWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilenameWidget</class>
   <extends>QWidget</extends>
   <header>FilenameWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AmarokDatabaseWidget</class>
   <extends>QWidget</extends>
   <header>AmarokDatabaseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WebBrowserWidget</class>
   <extends>QWidget</extends>
   <header>WebBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
